### DISCUSSION TITLE
不能正确的显示

### DISCUSSION THREAD

**Discussion by kieranwv** [Tencent/libpag#2589] [Q&A]
这个是项目中遇到的，pag 动效插件使用 cdn 或者 npm 安装都这样。

这是在一个轮播图上展示会员等级 logo 动效，其它 V2 - V9 的都是正常的，只有 V1 会这样。

![9ea9d00ee2e9c250f6b9bb70d4d47c3](https://github.com/user-attachments/assets/3c2d9bc4-3524-4635-ad49-c3f3ef3baf64)

**Comment by kieranwv** [comment]
目前是安卓手机出现，但是 IOS 是正常的

**Comment by jinwuwu001** [comment]
请问该问题是否还存在，若存在请提供一个可复现问题的demo.

**Comment by kevingpqi123** [comment]
由于长时间未收到进一步回复，暂时关闭此 Discussion。

如果后续有新的信息补充，欢迎重新打开或创建新的 Discussion。
